### ISSUE TITLE
what is the learning rate then finetuning all LLM parameters during SFT stage?

### ISSUE THREAD

**Issue by xmu-xiaoma666** [mbzuai-oryx/LLaVA-pp#5]
What is the learning rate then finetuning all LLM parameters during SFT stage?

**Comment by mmaaz60** [comment]
Hi @xmu-xiaoma666,

Thank you for your interest in our work. We use a learning rate of `2e-5` during full fine-tuning of both LLaMA-3 and Phi-3 based models. I hope it will help. Good Luck and let me know if you have any questions.

**Comment by mmaaz60** [comment]
Hi @xmu-xiaoma666,

We just added the [full finetuning script](https://github.com/mbzuai-oryx/LLaVA-pp/blob/main/scripts/LLaMA3-V_full_finetune.sh) that will reproduce our reported results. Good Luck and let us know if you have any questions.
